lgsm/functions/core_dl.sh

```diff
@@ -447,6 +447,47 @@ fn_update_function(){
 
 }
 
+# Function to download lastest github release
+# $1 github user / organisation
+# $2 repo name
+# $3 destination for download
+# $4 search string in releases (needed if there are more files that can be downloaded from the release pages)
+fn_dl_latest_release_github(){
+	local github_release_user="${1}"
```

Comment:
Is this indent space or tab? it is not clear on GitHub? The standard should be a `tab`